satpy/etc/composites/fci.yaml

```diff
@@ -246,6 +246,57 @@ composites:
       - flash_age
       - true_color_with_night_ir105
 
+  ### True Color with Fires
+  true_color_flames_with_night_ir105:
```

Comment:
Wasn't there a recommendation from the RGB workshop to name all day-only products with a `day` prefix?

Reply:
the fire overlay works day&night, so also the background composite is day&night, so we can probably leave it like this?